docs/usage/setup-azure-devops.md

```diff
@@ -1,21 +1,21 @@
 ---
-title: Azure DevOps Setup
+title: Azure DevOps setup
 description: How to setup Renovate for Azure DevOps
 ---
 
-# Azure DevOps Setup (for NPM/YARN)
+# Azure DevOps setup (for npm/Yarn)
 
 1. Create a brand new pipeline within Azure DevOps, and select your source
 
 ![Azure DevOps create new pipeline](assets/images/azure-devops-setup-1.png)
```

Comment:
I think some idention is needed to move the image to the numbered listing

Reply:
Yes, that was the problem, it's fixed now. 😄